bigflow/workflow.py

```diff
@@ -104,6 +104,7 @@ class Job(abc.ABC):
     id: str
     retries: int = 3
     retry_delay: float = 60
+    execution_timeout: int = None
```

Comment:
Provide the backward compatible default value explicitly so a user knows what is the default value.